src/plugins/conceptPlugin.js

```diff
@@ -95,14 +96,18 @@ export default function createConceptPlugin(options = {}) {
       undefined,
       { concept: true }
     );
-
     return render(
       <Notion
-        id={`notion_id_${id}`}
+        id={`notion_id_${id}_${locale}`}
         ariaLabel={t(locale, 'concept.showDescription')}
         title={title}
+        customCSS={css`
+          left: 0 !important;
+          margin-left: 0 !important;
+          width: 100% !important;
+        `}
         content={
-          <Fragment>
+          <>
```

Comment:
her igjen, trengs den egentlig å wrappes? ser det var der fra før

Reply:
Fjernet den forrige. Denne wrapper rundt to elementer

Reply:
stemmer, jeg så ikke på hele filen